src/runtime_src/core/common/query_requests.h

```diff
@@ -3073,6 +3074,15 @@ struct hwmon_sdm_fan_presence : request
   get(const device*) const = 0;
 };
 
+struct hwmon_sdm_mfg_date : request
```

Comment:
Please add a brief comment for  all new query requests.
Feel free to comment other ones too :-)

Reply:
Hi Soren, thank you for the review. I added comment for this query.